### DISCUSSION TITLE
umijs 在umirc或者app.tsx文件里面配置的logo不生效

### DISCUSSION THREAD

**Discussion by duzuimoye** [umijs/umi#11614] [Show and tell]
版本号如下：
<img width="428" alt="image" src="https://github.com/umijs/umi/assets/37163594/eef5c287-c067-468e-b36d-9fcb38c2e039">
在umirc.ts试过这种
<img width="749" alt="image" src="https://github.com/umijs/umi/assets/37163594/c7e9ebab-cbd8-4ee0-9503-a8a674b2b3c9">

也在app.ts试过
<img width="700" alt="image" src="https://github.com/umijs/umi/assets/37163594/7a3d0699-a7fb-4155-8acb-e135a44975dc">

目前这二种方法都尝试过 都不生效。**是不是版本的问题导致的？**

**Comment by fz6m** [comment]
用 `pnpm create umi demo` 创建一个新的 max 项目，logo 是好用的，自己排查下你的图片链接能不能正确访问，还有左上角的 logo 节点能不能加载出来吧。

如有问题，需要给一个最小复现。

**Reply by duzuimoye** [reply]
图片链接可以正常访问的[照片地址](https://s3.jn1.is.shanhe.com/kaiwu-static/97f961e6-ec01-47a4-ab4a-77993e666e74)、左上角的就是antd的icon
![image](https://github.com/umijs/umi/assets/37163594/739b8e0a-75a1-4445-9219-4748a50c9654)

**Reply by fz6m** [reply]
我创建了一个新的 max 项目后，在 `app.ts` 配置了你的图片，可以正常显示，如有问题，需要给一个最小复现。

```ts
// app.ts

export const layout = () => {
  return {
    logo: 'https://s3.jn1.is.shanhe.com/kaiwu-static/97f961e6-ec01-47a4-ab4a-77993e666e74',
  };
};
```
